pages/blog/escrow.js

```diff
@@ -0,0 +1,197 @@
+import Head from 'next/head'
+import Layout from '../../components/layout'
+
+export default function Blog() {
+  return (
+    <Layout>
+      <Head>
+        <title>Recover.ws - Loser Box to protect your item from loss</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <div className="escrow-container">
+        <h1 className="escrow-header">
+          Securing valuables with Escrow Smart Contracts
+        </h1>
+        <div>
+          <section className="blog__section">
+            <h2>What is a generic Escrow?</h2>
+            <p>
+              In traditional transactions, both the parties (payee and the
+              payer) are always concerned about foul play or mistrust. The payer
+              is usually concerned about the products/services they will receive
+              and payee for money being received. To fix this, an escrow is a
+              financial arrangement where a third party is responsible for
+              regulating and payment of funds which is required for multiple
+              parties involved in a given transaction. The third-party involved
+              only release the funds when all terms of the contract are
+              respected. It helps in the transaction being more secure as the
+              funds are kept in an escrow account which is managed by the
+              <a
+                className="blog_link"
+                href="https://en.wikipedia.org/wiki/Online_dispute_resolution"
+              >
+                arbitrator
+              </a>
+              . Also, it adds an extra level of trust both parties have for the
+              Escrow so chances of foul play are less. There are certain
+              obligations that need to be fulfilled before a payment is released
+              and in case the terms of the contract are not respected a dispute
+              can be raised. Escrow technology takes care of your transaction
+              without fraud or risk of losing money. It is always better to
+              trust Escrow than making a handshake deal which can result in foul
+              play afterward.
+            </p>
+          </section>
+          <section className="blog__section">
+            <h2>How escrow financial transaction works?</h2>
+            <p>
+              Initially, either the payee or the payer approaches Escrow which
+              acts as an arbitrator, the terms and conditions should be agreed
```

Comment:
Can you add 'upon' like "the terms and conditions should be agreed upon" @amitg01